lakefs_provider/operators/commit_operator.py

```diff
@@ -49,6 +92,26 @@ def execute(self, context: Dict[str, Any]) -> Any:
                       self.branch, self.repo)
 
         self.metadata.__setitem__("airflow_task_id", self.task_id)
+
+        # TODO(ariels): Configurably filter metadata keys.
+
+        cdd = self._get_current_dag_dict(context)
+        self.log.info("[DEBUG] cdd", cdd)
+
+        for k, template in self.metadata_templates.items():
+            try:
+                expanded = str(self.render_template(template, cdd))
+                self.metadata.__setitem__(self.metadata_key(k), expanded)
```

Comment:
```suggestion
                self.metadata[self.metadata_key(k)] = expanded
```
no?

Reply:
Not sure why we always do it this way.  @itaiad200 , perhaps you remember?